app/lib/shared/versions.dart

```diff
@@ -21,6 +21,7 @@ final RegExp runtimeVersionPattern = RegExp(r'^\d{4}\.\d{2}\.\d{2}$');
 /// Make sure that at least two versions are kept here as the next candidates
 /// when the version switch happens.
 const acceptedRuntimeVersions = <String>[
+  // TODO: Once this version is obsolete, remove Package.doNotAdvertise
```

Comment:
Why is this pending on runtimeVersion, isn't it more to do with deployment version? Hence, this deserves an entry in `CHANGELOG.md` that when versions older than next version doesn't exist then we can remove `doNotAdvertize`

Reply:
Updating changelog...